subprojects/core/src/test/groovy/org/gradle/api/internal/tasks/execution/DefaultEmptySourceTaskSkipperTest.groovy

```diff
@@ -225,9 +223,9 @@ class DefaultEmptySourceTaskSkipperTest extends Specification {
         0 * _
     }
 
-    def fingerprint(File... files) {
-        ImmutableSortedMap.<String, CurrentFileCollectionFingerprint> of(
-            "output", fingerprinter.fingerprint(TestFiles.fixed(files))
+    def snapshot(File... files) {
+        ImmutableSortedMap.<String, FileSystemSnapshot> of(
```

Comment:
```suggestion
        ImmutableSortedMap.<String, FileSystemSnapshot>of(
```

Nit: Space before `<>`.

Reply:
This is still IntelliJ misbehaving. I don't think it's worth fixing these.